### ISSUE TITLE
Rows count placeholder {TR} doesn't reflect on filtering

### ISSUE THREAD

**Issue by robertwebdev** [fooplugins/FooTable#525]
Hi,

the total number of rows placeholder `{TR}` doesn't reflect on filtering.
The format i'm using is `data-paging-count-format="{CP} / {TP} ({TR})"`.
Before filtering `{TR}` number is correct, but after some filtering it doesn't update to the current filterred rows count.

Also is it possible to place the rows count info anywhere on the page?

Rob

**Comment by jeffconklin** [comment]
> Also is it possible to place the rows count info anywhere on the page?

I would like to know the same but have never looked into it.

**Comment by steveush** [comment]
Hi,

I've fixed the `{TR}` placeholder in the develop branch and made some additional changes to paging to make setting a custom count label element a bit easier.

To hide the default count label use the below CSS in your page after you include the FooTable CSS files.

``` css
table.footable>tfoot>tr.footable-paging>td>span.label {
    display: none;
}
```

Then use the `postdraw.ft.table` event to get the current `formattedCount` string.

``` javascript
$('#table').footable({
    "on": {
        "postdraw.ft.table": function(e, ft){
            console.log(ft.use(FooTable.Paging).formattedCount);
        }
    }
});
```

Please grab the files from the [develop compiled folder](https://github.com/fooplugins/FooTable/tree/develop/compiled) for testing. If everything is working as expected I'll include this in the next release.

Thanks

**Comment by robertwebdev** [comment]
Thanks for the fix!
It was "must have" feature for me and I had a deadline so I must switched to DataTables. Sorry. No offence.
Anyway Footable is great!

**Comment by steveush** [comment]
This is now released in 3.1.2
